netdata.spec.in

```diff
@@ -132,12 +134,14 @@ rm -rf $RPM_BUILD_ROOT
 %config(noreplace) %{_sysconfdir}/%{name}/*.conf
 %config(noreplace) %{_sysconfdir}/%{name}/python.d/*.conf
 %dir %{_sysconfdir}/%{name}
-%{?with_systemd:%{_unitdir}/netdata.service}
+%{_sysconfdir}/logrotate.d/%{name}
```

Comment:
any file in `/etc` MUST be marked with `%config`! and as `%config` default is stupid, the line should be `%config(noreplace)`